app/assets/src/components/views/bulk_download/review_step.scss

```diff
@@ -61,5 +65,50 @@
       color: $medium-grey;
       margin-top: 6px;
     }
+
+    .loadingMessage {
+      color: $medium-grey;
+      display: flex;
+      align-items: center;
+
+      .icon {
+        margin-right: 4px;
+      }
+
+      .text {
+        @include font-body-xxs;
+      }
+    }
+
+    .errorMessage {
+      display: flex;
+
+      .icon {
+        fill: $error;
+        width: 18px;
+        height: 18px;
+        margin-right: 6px;
```

Comment:
Jenn would probably recommend using `$space-xxs` for margin-right ;)

Reply:
I'm planning on going through and re-doing the spacing values in a separate PR. Jenn sent me some new mocks that use the new values.